src/pyenphase/updaters/base.py

```diff
@@ -35,6 +37,10 @@ async def _json_probe_request(self, end_point: str) -> Any:
         response = await self._probe_request(end_point)
         return json_loads(end_point, response.content)
 
+    async def _add_common_property(self, property: str, property_value: Any) -> None:
```

Comment:
This doesn't need to be a co-routinue, nothing is being awaited here

Reply:
Tnx, will change